pkg/updater/gcs.go

```diff
@@ -526,9 +526,12 @@ func overallCell(result gcsResult) Cell {
 	return c
 }
 
-// ElapsedKey is the key for the test duration metric.
+// ElapsedKey is the key for the test results duration metric.
```

Comment:
nit: test-duration-minutes should be the test / target duration.

The one below, for consistency, should be "test-methods-duration-minutes", for the test methods / test results.

Reply:
Sorry to nitpick, I just think consistency with the codebase will be helpful in the long run.

Can this get updated to:

```
// ElapsedKey is the key for the target duration metric.
const ElapsedKey = "test-duration-minutes"

// TestMethodsElapsedKey is the key for the test results duration metric.
const TestMethodsElapsedKey = "test-methods-duration-minutes"
```

Thanks! Looks good to go after that.

Reply:
Fixed! @michelle192837